gradle.properties

```diff
@@ -15,3 +15,4 @@
 #
 
 org.curioswitch.curiostack.version = 0.0.143
+apiVersion = 0.0.9-SNAPSHOT
```

Comment:
Note: I renamed 'version' to 'apiVersion'. I didn't see anything using the existing property though.

Reply:
Ah - it was implicitly used by the java publishing :)